package.json

```diff
@@ -53,7 +53,7 @@
     "registry": "pnpm --filter @marigold/docs registry",
     "build-component-props": "pnpm --filter @marigold/docs build-component-props",
     "test": "jest --config jest.config.js",
-    "test:coverage": "jest --config jest.config.js --coverage --coverageReporters html",
+    "test:coverage": "jest --config jest.config.js --coverage --coverageReporters text",
```

Comment:
Why this change is necessary?

Reply:
because otherwise you can't see it? Or were is it? :D

Reply:
I just did this to have the report back, see uncovered lines, etc

Reply:
But now you just see it in the console where lines are uncovered but don't see the details right?

And with the flag `html` a directory is generated called coverage. The index.html you can open in the browser and have an interactive website with filter/search possibilities etc. Do you know that? So for me personally the html version is more handy. What do you think?

Reply:
I didn't know that xD